### DISCUSSION TITLE
Include PR URLs in the action output

### DISCUSSION THREAD

**Discussion by fnmarquez** [BetaHuhn/repo-file-sync-action#121] [Ideas] [answered]
Can you include an output that saves the pull requests URLs?
I need to use them in another step.

![image](https://user-images.githubusercontent.com/80843168/137321765-9ee5d9b2-adf3-4e75-bff4-f3d6341e4e23.png)

**Comment by BetaHuhn** [comment]
Yes, that should definitely be added.

It is already outputting the PR URL to `pull_request_url`, see here:

https://github.com/BetaHuhn/repo-file-sync-action/blob/eb3b6c638309861a170cf7374ac33d47ace120da/src/index.js#L172

The problem is that the variable is overwritten for each PR, so it will end up being the URL to the PR of the last repository.

Will add it when I have time this weekend, or feel free to create a PR yourself :)

**Reply by fnmarquez** [reply]
Thanks @BetaHuhn
Looking forward to this change.

**Comment by BetaHuhn** [comment, marked as answer]
Done, with [v1.15.0](https://github.com/betahuhn/repo-file-sync-action/releases/tag/v1.15.0) the action sets the `pull_request_urls` output to the URLs of each created PR.

More about this in the [README](https://github.com/BetaHuhn/repo-file-sync-action#outputs).

**Reply by Nef10** [reply]
@BetaHuhn Thanks! That was actually one of the things I was considering implementing myself :)

**Reply by BetaHuhn** [reply]
No problem :)

**Reply by fnmarquez** [reply]
Thanks! 👍
